### DISCUSSION TITLE
installation type : Minimalist

### DISCUSSION THREAD

**Discussion by Vonkje2000** [ampache/ampache#3134] [Q&A]
While installing ampache the other night it asked for the installation type. I know that I wil not use all the functions of ampache so I was intersted in de Minimalist install. When I was looking for more information about the minimalist installation I could not find anything. That is why i chose for the default installation. But my question is where can i find more information about the minimalist installation? 

One of the things I specifically was looking for is a list of the functions the minimalist installation has to compare with the default. But every information is welkom.

**Comment by mitchray** [comment]
That's a good question, it appears to make the following changes compared to the default install

Disables these:
- Ratings
 - Social
- Wanted (albums/artists)
- Channel
- Live stream
- Download
- Video

And sets the sidebar to collapsed mode, as well as default album/artist views as lists instead of grids

I'll add this info to the wiki

**Comment by lachlan-00** [comment]
https://public.ampache.dev/ uses the minimalist mode with additional settings enabled in the config (simple_user_mode to hide some elements and disabled use_auth and require_session) to allow a simple public site.
